src/styles/Navbar.module.css

```diff
@@ -0,0 +1,58 @@
+.panel-bg {
+  height: 5.938rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  flex-shrink: 0;
+}
```

Comment:
- [optional] please always try as much as possible to follow the complete design. your design is missing the user place holder. kindly fix that
yours

![Screenshot (843)_LI](https://user-images.githubusercontent.com/65298048/222184344-c5d8c2e0-a558-49e3-8145-9368823e8dad.jpg)
original design
![Screenshot (842)_LI](https://user-images.githubusercontent.com/65298048/222184556-57c38915-5c8d-420d-b7b1-c154a3e380a6.jpg)

Reply:
Hi @tufoinnkuo10, wow, thanks for the reminder. I forgot to add that feature.